### PR TITLE
fix: validates team tool url protocol

### PR DESCRIPTION
**Pull request by ana-yldio**
https://trello.com/c/Gxvd1zId/1038-keep-users-from-inputting-incomplete-urls-on-team-tools

### CODE REVIEW

File: packages/react-components/src/templates/__tests__/ToolModal.test.tsx [hunk 1]
```diff
@@ -39,6 +39,50 @@ it('renders default values into inputs', () => {
   `);
 });
 
+it('allows url with https protocol', () => {
+  const { getAllByRole, queryByText } = render(<ToolModal {...props} />, {
+    wrapper: StaticRouter,
+  });
+  const inputUrl = getAllByRole('textbox')[0];
```
Comment (kmaid) [inline packages/react-components/src/templates/__tests__/ToolModal.test.tsx:46]: Can we get by label text? [0] is quite brittle if we have to add another textbox

File: packages/validation/test/teams.test.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { TEAM_TOOL_URL } from '../src/teams';
+
+describe('ss', () => {
```
Comment (kmaid) [inline packages/validation/test/teams.test.ts:3]: ss?

**Review by kmaid — CHANGES_REQUESTED** [review]
looks good super minor things

File: packages/validation/test/teams.test.ts (hunk 2 above)
Comment (ana-yldio) [reply packages/validation/test/teams.test.ts]: sorry!

**Review by kmaid — APPROVED** [review]

**Pull request merged**